public/app/plugins/datasource/mssql/partials/config.html

```diff
@@ -29,6 +29,24 @@ <h3 class="page-heading">MSSQL Connection</h3>
 	</div>
 </div>
 
+<div class="gf-form-group">
+  <div class="gf-form max-width-30">
+    <span class="gf-form-label width-11">Max open connections</span>
+    <input type="number" min="1" class="gf-form-input" ng-model="ctrl.current.jsonData.maxOpenConns" placeholder="unlimited"></input>
```

Comment:
`min="1"` for all three configuration options wont allow to be set to zero once saved.

Reply:
0 is the same as the sql package's default, ie leaving it empty